src/wallet/rpcdump.cpp

```diff
@@ -510,3 +534,105 @@ UniValue dumpwallet(const UniValue& params, bool fHelp)
+                                          "{ \"type\":\"pubkey\", \"value\":\"<my public key>\", \"label\":\"example 1\", \"timestamp\":1455191480 } ]' true") +
+            HelpExampleCli("importmulti", "'[{ \"type\":\"pubkey\", \"value\":\"<my public key>\", \"label\":\"example 1\", \"timestamp\":1455191464 } ]' false") + HelpExampleRpc("importmulti", "[ { \"type\":\"privkey\", \"value\":\"<my private key>\" },"
+                                                                                                                                                                                                  "{ \"type\":\"pubkey\", \"value\":\"<my public key>\", \"label\":\"example 1\", \"timestamp\":1455191480 } ], true") +
+            HelpExampleRpc("importmulti", "'[{ \"type\":\"pubkey\", \"value\":\"<my public key>\", \"label\":\"example 1\", \"timestamp\":1455191464 } ]', false") +
+
+            "\nResponse is an array with the same size as the input that has the execution result :\n"
+            "  [ { \"result\": true | false } , ... ]\n");
+
+    if (!EnsureWalletIsAvailable(fHelp))
+        return NullUniValue;
+
+    if (params.size() == 1)
+        RPCTypeCheck(params, boost::assign::list_of(UniValue::VARR));
+    else
+        RPCTypeCheck(params, boost::assign::list_of(UniValue::VARR)(UniValue::VBOOL));
+
+    const UniValue& request = params[0];
+    bool fRescan = params.size() > 1 ? params[1].get_bool() : true;
+
+    UniValue response(UniValue::VARR);
+
+    LOCK2(cs_main, pwalletMain->cs_wallet);
+    EnsureWalletIsUnlocked();
+
+    bool fRunScan = false;
+    int64_t nLowestTimestamp = 0;
+
+    if (fRescan && chainActive.Tip())
+        nLowestTimestamp = chainActive.Tip()->GetBlockTime();
+    else
+        fRescan = false;
+
+
+    BOOST_FOREACH (const UniValue& data, request.getValues()) {
+        UniValue result = UniValue(UniValue::VOBJ);
+        try {
+            if (nLowestTimestamp > 0) {
+                int64_t timestamp = data.exists("timestamp") ? data["timestamp"].get_int64() : 0;
+
+                if (timestamp < nLowestTimestamp)
+                    nLowestTimestamp = timestamp;
+            }
+
+            const string& strType = data["type"].get_str();
+            const string& strValue = data["value"].get_str();
+            string strLabel = data.exists("label") ? data["label"].get_str() : "";
+
+            if (strType == "privkey")
+                ImportPrivateKey(strValue, strLabel);
+            else if (strType == "pubkey")
+                ImportPublicKey(strValue, strLabel);
+            else if (strType == "address") {
+                bool fP2SH = data.exists("p2sh") ? data["p2sh"].get_bool() : false;
+                ImportAddressKey(strValue, strLabel, fP2SH);
+            } else
+                throw;
+
+            result.pushKV("result", UniValue(true));
+            fRunScan = true;
+        } catch (...) {
```

Comment:
IMO should catch the `JSONRPCError` and be included on the response, something like:
```bash
output: [
  {
    "result": true
  },
  {
    "result": false,
    "error": {
      "code": -5,
      "message": "Invalid Bitcoin address"
    }
  }
]
```
and catch other exceptions as "missing required fields". Also maybe changing from "result" to "success".